samples/unity-of-bugs/Assets/Scripts/SentryOptionConfiguration.cs

```diff
@@ -6,10 +6,19 @@ public class SentryOptionConfiguration : SentryOptionsConfiguration
 {
     public override void Configure(SentryUnityOptions options)
     {
-#if UNITY_ANDROID
-        options.SampleRate = 0.1f;
-#elif UNITY_IOS
-        options.SampleRate = 1.0f;
+        // Here you can programmatically modify the Sentry option properties used for the SDK's initialization
+
+#if UNITY_ANDROID || UNITY_IOS
```

Comment:
Should we just make the C# layer initialize the native layer by default? 

And make this "Native inits first" thing be opt-in? It seems still hard for folks to understand what's going on, even with the comments below. (I don't expect a beginner to get it)

Reply:
It would be simpler but we would lose on the ability to catch native exceptions happening at startup. Is that worth it?

Reply:
I can see a point for: If you've got plugins that might crash the app before the Unity game starts you can opt-in the native first init and still have full coverage. 
If you don't then are those native crashes/exceptions be even actionable for you?

Reply:
> I can see a point for: If you've got plugins that might crash the app before the Unity game starts you can opt-in the native first init and still have full coverage. 

Wouldn't that mean we still need to keep the buildtime native configuration generator (which I understand Bruno was suggesting we could remove)?

> If you don't then are those native crashes/exceptions be even actionable for you?

Even the information that there are startup crashes is valuable IMO. 

---- 
Overall, I wouldn't change this - it does provide value (startup crashes) even though it does make things a little more complicated IF users need to adapt these settings programmatically